.github/workflows/tests.yml

```diff
@@ -11,11 +11,17 @@ jobs:
 
     strategy:
       matrix:
-        php: ['7.4', '8.0', '8.1', '8.2']
-        symfony: ['4', '5', '6']
+        php: ["7.4", "8.0", "8.1", "8.2"]
```

Comment:
Please revert these back to single quotes.

Reply:
sorry, it's fixed :)